### PR TITLE
First commit with styling of text, buttons and layouts

### PR DESCRIPTION
**Pull request by CalexisGonzalez**
First iteration with theme for the whole app implemented, including buttons, text and colors.

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -19,4 +19,14 @@
     <dimen name="mainscreen_settings_vertical_margin">16dp</dimen>
     <dimen name="mainscreen_logout_left_margin">72dp</dimen>
     <dimen name="mainscreen_logout_bottom_margin">95dp</dimen>
+    <dimen name="presentation_logo_marginStart">176dp</dimen>
+    <dimen name="presentation_logo_maginTop">308dp</dimen>
+    <item name="logo_anim_value_from" format="float" type="dimen">0.0</item>
```
Comment (SartiMau) [inline app/src/main/res/values/dimens.xml]: you can pass this to floats.xml

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -6,6 +6,23 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:fontFamily">@font/text_font</item>
+        <item name="android:colorSecondary">@color/colorSecondary</item>
+        <item name="android:windowBackground">@color/colorBackground</item>
+        <item name="android:textSize">15sp</item>
```
Comment (SartiMau) [inline app/src/main/res/values/styles.xml]: no hardcode pls

File: app/src/main/res/layout/activity_user_settings.xml [hunk 3]
```diff
@@ -88,9 +88,10 @@
             android:visibility="visible" />
     </TableRow>
 
-    <TableRow
+    <LinearLayout
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:orientation="horizontal"
         android:layout_marginHorizontal="@dimen/signup_activity_horizontal_margin"
         android:layout_marginVertical="@dimen/signup_activity_vertical_margin">
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_user_settings.xml]: remove unnecessary extra line pls

File: app/src/main/res/layout/activity_sign_up.xml [hunk 4]
```diff
@@ -103,28 +103,33 @@
             android:inputType="textPersonName" />
     </TableRow>
 
-    <TableRow
+    <LinearLayout
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:orientation="horizontal"
         android:layout_marginHorizontal="@dimen/signup_activity_horizontal_margin"
         android:layout_marginVertical="@dimen/signup_activity_vertical_margin">
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_sign_up.xml]: remove unnecessary extra line

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_sign_up.xml (hunk 4 above)
Comment (CalexisGonzalez) [reply app/src/main/res/layout/activity_sign_up.xml]: Done

File: app/src/main/res/layout/activity_user_settings.xml (hunk 3 above)
Comment (CalexisGonzalez) [reply app/src/main/res/layout/activity_user_settings.xml]: Done

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (CalexisGonzalez) [reply app/src/main/res/values/styles.xml]: Done

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (CalexisGonzalez) [reply app/src/main/res/values/dimens.xml]: Done

**Review by SartiMau — APPROVED** [review]

**Pull request merged**